addons/website_sale_stock/static/tests/tours/website_sale_stock_product_configurator.js

```diff
@@ -23,12 +23,20 @@ registry
             // Assert that it's impossible to add less than 1 product (only for the main product).
             configuratorTourUtils.setProductQuantity("Main product", 0),
             configuratorTourUtils.assertProductQuantity("Main product", 1),
-            configuratorTourUtils.decreaseProductQuantity("Main product"),
+            {
+                content: "check that decrease button is disabled",
+                trigger: `.modal button[name=sale_product_configurator_quantity_button_minus]:disabled`,
+                allowDisabled: true,
+            },
             configuratorTourUtils.assertProductQuantity("Main product", 1),
             // Assert that it's impossible to add more products than available.
             configuratorTourUtils.setProductQuantity("Main product", 20),
             configuratorTourUtils.assertProductQuantity("Main product", 10),
-            configuratorTourUtils.increaseProductQuantity("Main product"),
```

Comment:
Could you remove `allowDisabled: true,` from the `increaseProductQuantity` function? It shouldn't be needed anymore.

Reply:
(We may also want to update said function to rely on the name attribute, i.e. `sale_product_configurator_quantity_button_plus`, instead of on the icon, but that's not really related to this PR)

Reply:
allowDiabled will totally disapear in a next PR ... it's the goal :) ... Don't worry.